src/lib.rs

```diff
@@ -571,18 +616,15 @@ pub fn init(options: &InitOptions) -> Result<(ZWaveManager, mpsc::Receiver<ZWave
     let (mut zwave_manager, rx) = ZWaveManager::new(manager);
     try!(zwave_manager.add_watcher());
 
-    let device = match options.device {
-        Some(ref device) => device as &str,
-        _ => try!(get_default_device())
-    };
-
-    try!(
-        fs::File::open(device).map_err(|err| Error::CannotReadDevice(device.to_string(), err))
-    );
-
-    //println!("found device {}", device);
+    let devices = options.devices.clone().unwrap_or(get_default_devices());
```

Comment:
I'm sure we can find another way because we merely need a reference here; but in this case we don't really mind as "devices" is small, so let's move on.